File: ios/Classes/Keys.swift
import Foundation

// TODO: Cleanup this file later when all commented methods are implemented
struct Keys {
    
    struct Methods{
        static let initialize = "initialize"
        static let identify = "identify"
        static let clearIdentify = "clearIdentify"
        static let track = "track"
        static let screen = "screen"
        static let setDeviceAttributes = "setDeviceAttributes"
        static let setProfileAttributes = "setProfileAttributes"
        static let registerDeviceToken = "registerDeviceToken"
        static let trackMetric = "trackMetric"
        static let dismissMessage = "dismissMessage"
    }
    
    struct Tracking {
        static let userId = "userId"
        static let traits = "traits"
        static let eventName = "eventName"
        static let token = "token"
        static let deliveryId = "deliveryId"
        static let deliveryToken = "deliveryToken"
        static let metricEvent = "metricEvent"
        
        static let name = "name"
        static let properties = "properties"
    }
    
    struct Environment{
        static let siteId = "siteId"
        static let apiKey = "apiKey"
        static let region = "region"
        static let enableInApp = "enableInApp"
    }
    
}

File: android/src/main/kotlin/io/customer/customer_io/constant/Keys.kt
package io.customer.customer_io.constant

// TODO: Cleanup this file later when all commented methods are implemented
internal object Keys {

    object Methods {
        const val INITIALIZE = "initialize"
        const val IDENTIFY = "identify"
        const val CLEAR_IDENTIFY = "clearIdentify"
        const val TRACK = "track"
        const val SCREEN = "screen"
        const val SET_DEVICE_ATTRIBUTES = "setDeviceAttributes"
        const val SET_PROFILE_ATTRIBUTES = "setProfileAttributes"
        const val REGISTER_DEVICE_TOKEN = "registerDeviceToken"
        const val TRACK_METRIC = "trackMetric"
        const val ON_MESSAGE_RECEIVED = "onMessageReceived"
        const val DISMISS_MESSAGE = "dismissMessage"
    }

    object Tracking {
        const val USER_ID = "userId"
        const val TRAITS = "traits"
        const val EVENT_NAME = "eventName"
        const val TOKEN = "token"
        const val DELIVERY_ID = "deliveryId"
        const val DELIVERY_TOKEN = "deliveryToken"
        const val METRIC_EVENT = "metricEvent"

        const val NAME = "name"
        const val PROPERTIES = "properties"
        const val TITLE = "title"
    }
}
